### PR TITLE
Adds the compact mode

### PR DESCRIPTION
**Pull request by weeman1337**
* Resolves: #587
* Target version: master 

### Summary

Adds a compact mode toggle:

**standard view and toggle**
image

**compact mode**
image

### Checklist

- [X] Code is properly formatted
- [X] Sign-off message is added to all commits
- [X] Tests (unit, integration, api and/or acceptance) are included (no frontend tests, yet)
- [X] Documentation (manuals or wiki) has been updated or is not required

### Possible follow ups

* introduce some JS in the core, that allows storing user settings
* update deck to persist the compact mode state

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -338,7 +348,7 @@ input.input-inline {
 	flex-direction: column;
 
 	h3 {
-		padding: 10px 10px 8px;
+		padding: 10px 10px 8px;;
```
Comment (juliushaertl) [inline css/style.scss]: Duplicate ;

**Comment by Mannshoch** [comment]
Please insert the Color into the small view. Maybe as small line or a little Box without Text (as tool-tip)
grafik

**Comment by juliushaertl** [comment]
Good point @Mannshoch mind to open an issue so we don't forget about this, since it is already merged?

**Comment by weeman1337** [comment]
@Mannshoch with color you mean the tag colors, right?

**Comment by Mannshoch** [comment]
@weeman1337 yes

**Comment by Mannshoch** [comment]
If you asking because my comment on  https://github.com/nextcloud/deck/issues/15#issuecomment-413128639 that's why I set this colors in this Mockup in the lower center.

**Pull request merged**